packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/MultiFrameViewer/FrameCarousel.js

```diff
@@ -48,14 +48,14 @@ class FrameCarousel extends React.Component {
   render () {
     const locations = this.props.subject.locations.map(location => ({ 'url': location['image/jpeg'] }))
     const locationElements = locations.map((location, index) => {
-      const currentFrameIndex = parseInt(this.props.subject.metadata.default_frame)
-      const currentActive = currentFrameIndex === index
+      // const currentFrameIndex = parseInt(this.props.subject.metadata.default_frame)
+      // const currentActive = currentFrameIndex === index
```

Comment:
Should these be removed or are they temporarily commented? If the latter, could you add a TODO comment?